e2e/Dockerfile.jenkins

```
@@ -1,7 +1,5 @@
-FROM ubuntu:20.04
-
-# Setting bash as the default shell
-RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+ARG NODE_VERSION=16.15.0
+FROM node:$NODE_VERSION-buster
```

it's even possible to avoid `-buster` see https://hub.docker.com/layers/library/node/16.15.0/images/sha256-2d085d25d573a58266319362f32e978f94e06214a62fae15c02d4c4f0567b64f?context=explore


```suggestion
FROM node:$NODE_VERSION
```